ui/config/config.js

```diff
@@ -55,7 +55,11 @@ class CactbotConfigurator {
       'jobs': [],
     };
     this.configOptions = configOptions;
-    this.lang = configOptions.Language || 'en';
+    let shortLocale = configOptions.ShortLocale;
+    const supportedLanguage = ['en', 'de', 'fr', 'ja', 'cn', 'ko'];
+    if (supportedLanguage.indexOf(shortLocale) == -1)
```

Comment:
Can you put this code to validate the locale into one of the valid languages up in user_config.js? That way anybody else using it can just use variable directly without also having to do this.